packages/fetch-http-handler/src/fetch-http-handler.ts

```diff
@@ -130,4 +130,16 @@ export class FetchHttpHandler implements HttpHandler {
     }
     return Promise.race(raceOfPromises);
   }
+
+  updateHttpClientConfig(key: keyof FetchHttpHandlerConfig, value: FetchHttpHandlerConfig[typeof key]): void {
+    this.config = undefined;
+    this.configProvider = this.configProvider.then((resolveConfig) => {
```

Comment:
the resolved value of configProvider should be named `config`, because `resolveConfig` sounds like an action (method).